AppKit/CPView.j

```diff
@@ -3511,7 +3703,7 @@ var CPViewAutoresizingMaskKey       = @"CPViewAutoresizingMask",
         [self _decodeThemeObjectsWithCoder:aCoder];
 
         [self setAppearance:[aCoder decodeObjectForKey:CPViewAppearanceKey]];
-
+        
```

Comment:
If that were the case, this change would not have been made in the commit. Are you sure you have "Including whitespace-only lines" checked in the Editor prefs? Either that is not checked or there's a bug in Xcode.